src/airflow/main.py

```diff
@@ -572,6 +572,48 @@ def urban_better_data_from_air_beam():
     bigquery_data.to_csv("urban_better_air_beam_bigquery_data.csv", index=False)
 
 
+def urban_better_data_from_bigquery():
+    from airqo_etl_utils.urban_better_utils import UrbanBetterUtils
+
+    bigquery_api = BigQueryApi()
+    data = bigquery_api.query_data(
+        start_date_time="2022-01-01T00:00:00Z",
```

Comment:
please shed some light on these "default dates"? What is the context behind them? Are they even default dates? @NoahNsimbe

Reply:
The `main.py` file is a playground for functions in the package. @Baalmart 
https://github.com/airqo-platform/AirQo-api/tree/staging/src/airflow/airqo_etl_utils